index.html

```diff
@@ -114,13 +124,143 @@ <h2>Introduction</h2>
+      </p>
+      <p>
+        Section 2.3 of [[TTML2]] specifies conventions used in the [[XML]] representation of elements; and
+        Sections 6.2 and 8.2 of [[TTML2]] specify conventions used when specifying the syntax of attribute values.
+      </p>
+      <p>
+        All content of this specification that is not explicitly marked as non-normative is considered to be normative.
+        If a section or appendix header contains the expression "non-normative", then the entirety of the section or appendix is considered non-normative.
+      </p>
+      <p>
+        This specification uses Feature designations as defined in Appendices E at [[TTML2]]:
+        when making reference to content conformance, these designations refer to the syntactic expression or the semantic capability associated with each designated Feature; and
+        when making reference to processor conformance, these designations refer to processing requirements associated with each designated Feature.
+        If the name of an element referenced in this specification is not namespace qualified, then the TT namespace applies (see <a href="#namespaces">9.3 Namespaces</a>.)
+      </p>
+    </section>
+
+    <section id="definitions">
+      <h2>Definitions</h2>
+      <p>
+        The following terms are used in this proposal:
+      </p>
+      <p>
+        <dfn>Audio description</dfn>	An audio rendition of a <a>Description</a> or a set of <a>Descriptions</a>.
+      </p>
+      <p>
+        <dfn data-lt="Description|Descriptions">Description</dfn>  A set of words that describe an aspect of the programme presentation, suitable for rendering into audio by means of vocalisation and recording or used as a <a>Text Alternative</a> source for text to speech translation.
+      </p>
+      <p>
+        <dfn data-lt="Document Instance|Document Instances" data-cite="ttml2#terms-document-instance">Document Instance</dfn> As defined by [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-document-interchange-context">Document Interchange Context</dfn> As defined by [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-document-processing-context">Document Processing Context</dfn>  See Section 2.2 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-feature">Feature</dfn>  See Section 2.2 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-intermediate-synchronic-document">Intermediate Synchronic Document</dfn> See Section 11.3.1.3 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#style-value-lwsp">Linear White-Space</dfn> See Section 2.3 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-presentation-processor">Presentation processor</dfn> See Section 2.2 at [[TTML2]].
+      </p>
+      <p>
+        <dfn data-cite="ttml2#terms-processor">Processor</dfn>  Either a Presentation processor or a Transformation processor.
+      </p>
```

Comment:
I would prefer if we defined one (or more) classes of DAPT Processors, for example DAPT Authoring Tool, DAPT Audio Recorder/Renderer, DAPT-to-IMSC Convertor. For each of them, we could say if they are "TTML2 presentation processors" or "TTML2 transformation processors" for those who are interested in digging into what that means, but in practice a DAPT implementor should not need to care about it.

Reply:
I'm happy to do that if it's easier to read - I suggest we open a separate issue and PR to make that change. The formal specification is important, but we should promote readability.

Reply:
done https://github.com/w3c/dapt/issues/44